arbnode/dataposter/data_poster.go

```diff
@@ -1430,6 +1431,7 @@ var TestDataPosterConfig = DataPosterConfig{
 	LegacyStorageEncoding:  false,
 	ExternalSigner:         ExternalSignerCfg{Method: "eth_signTransaction", InsecureSkipVerify: true},
 	MaxFeeCapFormula:       "((BacklogOfBatches * UrgencyGWei) ** 2) + ((ElapsedTime/ElapsedTimeBase) ** 2) * ElapsedTimeImportance + TargetPriceGWei",
+	MaxBaseFee:             5000000000,
```

Comment:
Should we hardcode this value?

Reply:
This is just default test config value, it can be overwritten